src/app/organisms/room/RoomTimeline.tsx

```diff
@@ -1280,9 +1346,13 @@ export function RoomTimeline({ room, eventId, roomInputRef, editor }: RoomTimeli
           relations={hasReactions ? reactionRelations : undefined}
           onUserClick={handleUserClick}
           onUsernameClick={handleUsernameClick}
-          onReplyClick={handleReplyClick}
+          onReplyClick={(evt: MouseEvent) => handleReplyId(evt.currentTarget.getAttribute('data-event-id'))}
           onReactionToggle={handleReactionToggle}
           onEditId={handleEdit}
+          onTouchStart={(evt: TouchEvent) => onTouchStart(evt, mEvent.getId())}
+          onTouchMove={(evt: TouchEvent) => onTouchMove(evt, mEvent.getId())}
+          onTouchEnd={onTouchEnd}
+          style={isTouchingSide && mEvent.getId() == swipingId ? { transform: `translateX(${clamp(sideMoved - sideMovedInit, -window.innerWidth, 0)}px)` } : { transition: "all .25s ease" }}
```

Comment:
Could we maybe define these styles with vanilla-extract css module?

Reply:
Put into `MessageBase` in `style.css.ts` https://github.com/cinnyapp/cinny/pull/1647/commits/67077d9a3f79145a7a7162cd084547953baa9358